postgres-appliance/spilok8s.yaml

```diff
@@ -44,16 +28,44 @@ spec:
         volumeMounts:
         - mountPath: /home/postgres/pgdata
           name: pgdata
-        - mountPath: /etc/wal-e
-          name: wale-config
-          readOnly: true
+        env:
+        - name: ETCD_HOST
+          value: 'etcd.default.svc.cluster.local:2379' # where is your etcd?
```

Comment:
Services like ETCD should run as [kubernetes services](http://kubernetes.io/docs/user-guide/services/) correct? If so, could we leverage [label selectors](http://kubernetes.io/docs/user-guide/labels/#label-selectors) to connect to the ETCD cluster?